runtime/lua/vim/lsp/log.lua

```diff
@@ -74,6 +99,8 @@ end
 -- interfere with iterating the levels
 vim.tbl_add_reverse_lookup(log.levels)
 
+--- Sets the current log level.
+--@param level (string or number) One of `vim.lsp.log.levels`
```

Comment:
Should probably be a tag (do we have a tag for the levels?)

Reply:
Not yet, I can add one. In this PR?

Reply:
Yeah. You can hard code some things into the docs if you put them above the generated code.  Or maybe at the top w/ a `@brief`? I would have to experiment with that to check though.

Reply:
Hmm, I'd rather not add stuff manually to the doc output, it could easily disappear if someone doesn't look too hard at a commit's changes.

Is there maybe already some easy way to get "module documentation" to show in the Vimdoc output? (cc @KillTheMule)

Reply:
cc @justinmk:

> Is there maybe already some easy way to get "module documentation" to show in the Vimdoc output?